RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py

```diff
@@ -19,8 +19,8 @@
              srFlags = cms.InputTag("ecalDigis"),
              qualityTests = cms.VPSet(
                   cms.PSet(
-                  name = cms.string("PFRecHitQTestECALMultiThreshold"),
-                  thresholds = particle_flow_zero_suppression_ECAL.thresholds
+                  name = cms.string("PFRecHitQTestThreshold"),
+                  threshold = cms.double(0.08)
```

Comment:
I didn't realize that we have an issue in the barrel as well.
In case I missed something, please point me to some plots that quantify degradation in the barrel.